index.html

```diff
@@ -424,7 +424,67 @@ <h3>Terminal</h3>
                     </ul>
                 </div>
             </section>
+            <section class="contactme">
+                <div class="contactme__content">
+                    <div class="contactme__details">
+                        <p>
+                            I'm always <br> interested in hearing <br>
+                            about new projects, <br>
+                            so if you'd like to <br>
+                            chat please get in touch.
+                        </p>
+                    </div>
+                    
+                </div>
+                <form action="https://formspree.io/f/mzbqzrnz" method="POST" class="contactme__content">
```

Comment:
- [ ] You did a great job implementing the `Formspree` service as requested by the previous reviewer! 💯 However, at the moment, your form is only getting the data from the `<textarea>` element.

![image](https://user-images.githubusercontent.com/77069011/212968012-c3d0cf79-74b1-4759-b09d-b4fad62cfb30.png)

This happens because forms only collect data on submission from elements that have `name` attributes. Notice that your `<textarea>` has `name="contactme"` while your `name` and `email` inputs don't. Please consider adding this attribute to them! 🤓 🔧